.github/workflows/release-chocolatey.yml

```diff
@@ -0,0 +1,42 @@
+name: Release Chocolatey Package
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
```

Comment:
we should avoid manual releases, I guess it is here just for testing?

how about doing it the same as with brew -> that we release whatever is new once a day? since we have a lot of automation, might be that we have minor release 2-3 times a day, so we should avoid hitting chocolatey community, like we did with brew

Reply:
Yeah. I have removed it.